.github/workflows/build_and_push_sdk_packages.yaml

```diff
@@ -0,0 +1,32 @@
+name: SDK Publishing
```

Comment:
Why not combine the package and push in one workflow?

Reply:
Because we may want to package but not publish (internal releases, etc). Workflow triggers allow having both workflows separate (if it works as intended) but also create dependencies.

Reply:
We'll package package and push to PyPI in a single workflow.
We'll investigate how to keep the pyproject version in sync with the git tag for releases.